scripts/exhafs_product.sh

```diff
@@ -271,6 +272,36 @@ fi
 
 fi
 
+#===============================================================================
+# generate nhc products
+mkdir -p ${DATA}/nhc_products
+cd ${DATA}/nhc_products
+
+cp -p ${NHCPRODUCTSEXEC} ./hafs_nhc_products.x
+cp ${COMhafs}/storm1.holdvars.txt .
+ln -sf ${COMhafs}/${trk_atcfunix} fort.20
+
+set +e
+set -o pipefail
+time ./hafs_nhc_products.x  2>&1 | tee ./hafs_nhc_products.out
+set +o pipefail
+set -e
+
+short=${out_prefix}.grib.stats.short
+afos=${out_prefix}.afos
+tpc=${out_prefix}.stats.tpc
```

Comment:
@LinZhu-NOAA, could you update these three lines into:
short=${out_prefix}.hafs.grib.stats.short
afos=${out_prefix}.hafs.afos
tpc=${out_prefix}.hafs.stats.tpc

Currently, all HAFS products in com dir have the naming convention of "${out_prefix}.hafs.*". In the future, if we come up new updated convention, we can further update this. Thanks!